billing/urls.py

```diff
@@ -1,11 +1,17 @@
-from django.conf.urls import url
+
+from django.urls import path
 from rest_framework import routers
 
-from .views import AccountView, CreditCardViewSet
+from .views import AccountView, CreditCardViewSet, pay_debt
 
 router = routers.SimpleRouter(trailing_slash=False)
-router.register(r'account/credit-cards', CreditCardViewSet, basename='billing_creditcard')
+router.register(
+    r'account/credit-cards',
+    CreditCardViewSet,
+    basename='billing_creditcard'
+)
 
 urlpatterns = router.urls + [
-    url(r'^account$', AccountView.as_view(), name='billing_account'),
+    path('account', AccountView.as_view(), name='billing_account'),
+    path('pay/debt', pay_debt, name='pay_debt'),
```

Comment:
I think the url/name should be more precise: pay_open_invoices_with_registered_credit_cards ?

Reply:
Done. Would you change the regular expression too?

Reply:
The endpoint `pay/debt`

Reply:
yes that's the most important thing to change (its the public api of all this), 
changing the internals to be clearer is also important (but slightly less)

Reply:
Done